master/docs/manual/cfg-workers-docker.rst

```diff
@@ -285,9 +285,15 @@ In addition to the arguments available for any :ref:`Latent-Workers`, :class:`Hy
 
 .. code-block:: python
 
-    import requests, urlparse
+    try:
```

Comment:
I think as buildbot requires future, it is safe to just document with

    from future.moves.urllib.parse import urlparse

or the sake of simplicity